src/main/java/io/github/nucleuspowered/nucleus/internal/services/datastore/UserService.java

```diff
@@ -413,4 +413,14 @@ public boolean jailOnNextLogin() {
     public void setJailOnNextLogin(boolean set) {
         config.setJailOffline(!user.isOnline() && set);
     }
+
+    @Override
+    public Map<String, Long> getKitLastUsedTime() {
+        return config.getKitLastUsedTime();
+    }
+
+    @Override
+    public void setKitLastUsedTime(Map<String, Long> kitLastUsedTime) {
```

Comment:
As well as being able to replace the whole map, consider a method that just adds the kit, so `public void setKitLastUsedTime(String kit, Long time)`.

What time unit is the used time? Is it from `Instant.getEpochMilli()?`